mcs/build/tests.make

```diff
@@ -157,6 +157,9 @@ $(test_makefrag): $(test_response)
 
 -include $(test_makefrag)
 
+build-test-lib: $(test_lib)
```

Comment:
More precisely, I'm building the test lib and using an external runner - which means I can't use make check as it will fail once it tries to run.

I want to keep the knowledge of how to build the test suite with mono as a ton of flags are passed to csc.